From 6067dbdf0fe3c2d83055c43edf35cddf277e7236 Mon Sep 17 00:00:00 2001
From: Tomas Bzatek <tbzatek@redhat.com>
Date: Fri, 10 Nov 2023 14:09:20 +0100
Subject: [PATCH] tests: Remove unreliable nvme attribute checks

Some attributes like this health information data read counters
are just too volatile to be tested as there might be some other
processes probing the device causing the statistics counters to increase.
---
 tests/nvme_test.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tests/nvme_test.py b/tests/nvme_test.py
index 3c7c0a29..f31de546 100644
--- a/tests/nvme_test.py
+++ b/tests/nvme_test.py
@@ -157,14 +157,12 @@ def test_smart_log(self):
         self.assertEqual(log.critical_temp_time, 0)
         self.assertEqual(log.ctrl_busy_time, 0)
         self.assertEqual(log.media_errors, 0)
-        # self.assertEqual(log.num_err_log_entries, 0)
         self.assertEqual(log.percent_used, 0)
         self.assertEqual(log.power_cycles, 0)
         self.assertEqual(log.power_on_hours, 0)
         self.assertEqual(log.spare_thresh, 0)
         self.assertEqual(log.temp_sensors, [0, 0, 0, 0, 0, 0, 0, 0])
         self.assertEqual(log.temperature, 0)
-        self.assertGreater(log.total_data_read, 1)
         self.assertEqual(log.unsafe_shutdowns, 0)
         self.assertEqual(log.warning_temp_time, 0)
         self.assertEqual(log.wctemp, 0)
